frontend/packages/ux-editor/src/components/config/editModal/EditCodeList.test.tsx

```diff
@@ -31,6 +31,16 @@ describe('EditCodeList', () => {
     await act(() => user.click(screen.getByRole('option', { name: 'test-1' })));
     await waitFor(() => expect(handleComponentChangeMock).toHaveBeenCalled());
   });
+
+  it('displays selected option list on component render', async () => {
```

Comment:
Just a small suggestion. 😊 

```suggestion
  it('should render the selected option list item upon component initialization', async () => {
```

Reply:
I agree, I like that better :)